### PR TITLE
Removing min_profile

### PR DESCRIPTION
**Pull request by anusapan**
---
This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

Thank you for contributing to the IoT extension!

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] If introducing new functionality or modified behavior, are they backed by unit and integration tests?
- [ ] In the same context as above are command names and their parameter definitions accurate? Do help docs have sufficient content?
- [ ] Have **all** unit **and** integration tests passed locally? i.e. `pytest -s <project root>/azext_iot/tests`
- [ ] Have static checks passed using the .pylintrc and .flake8 rules? Look at the CI scripts for example usage.

### CODE REVIEW

File: azext_iot/azext_metadata.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
     "azext.minCliCoreVersion": "2.0.24",
-    "version": "0.7.0"
+    "azext.maxCliCoreVersion": "2.0.66",
```
Comment (limingu) [inline azext_iot/azext_metadata.json]: You don't need to add azext.maxCliCoreVersion": "2.0.66" if you removed min_profile.

Comment (c-ryan-k) [reply azext_iot/azext_metadata.json]: @limingu - so as long as minProfile is gone, we don't need to set a max version, and therefore won't need to release again when 2.0.67 is released?

Comment (anusapan) [reply azext_iot/azext_metadata.json]: I think we have to change the azext.minCliCoreVersion": "2.0.67" once 2.0.67 is released, Correct me if I am wrong

**Comment by c-ryan-k** [comment]
@digimaun - any input on the impact of removing the 'min_profile'? This is something I'm not too familiar with.

File: azext_iot/azext_metadata.json (hunk 1 above)
Comment (limingu) [reply azext_iot/azext_metadata.json]: Did iot really need min_profile? If not, removing min_profile is the easiest way. One publish is good enough.

If iot needed min_profile, you need to add the max version.

**Pull request merged**